lib/actions/ui.js

```diff
@@ -137,6 +137,10 @@ export function moveTo (i) {
 }
 
 export function showPreferences () {
+  var editorFallback = 'nano';
```

Comment:
`const editorFallback = process.platform === 'win32' ? 'notepad' : 'nano';`
or use `let` instead of `var` if you'd rather have the if statement.